ui/src/common/components/SearchFeedback/SearchFeedback.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, CSSProperties } from 'react';
+import { Card, Modal, Form, Input, message, Typography } from 'antd';
+import { useForm, useWatch } from 'antd/lib/form/Form';
+import { getClientId, trackEvent } from '../../../tracker';
+
+const { TextArea } = Input;
+
+const SearchFeedback = ({ style }: { style?: CSSProperties }) => {
+  const [feedbackModalVisible, setFeedbackModalVisible] = useState(false);
+  const [form] = useForm();
+
+  const question = useWatch('question', form);
+
+  const handleSubmit = async () => {
+    try {
+      const values = form.getFieldsValue();
+
+      const response = await fetch('/ai/v1/search-feedback', {
```

Comment:
I see you are using fetch. We mainly use axios and HttpClientWrapper, is it possible to utilize it for consistency e.g https://github.com/inspirehep/inspirehep/blob/master/ui/src/common/http.ts#L145?